contracts/pcv/angle/AngleEuroRedeemer.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./IStableMaster.sol";
+import "../../external/Decimal.sol";
+import "../../oracle/IOracle.sol";
+import "../../refs/CoreRef.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IMakerPSM {
+    function sellGem(address, uint256) external;
+}
+
+/// @title Contract to redeem agEUR for FEI and DAI
+/// @author eswak
+contract AngleEuroRedeemer is CoreRef {
```

Comment:
This is a fair amount of Solidity to be shipping without tests/integration tests. 

Is it practical to write an integration test that validates we receive the expected amount of DAI out for an amount of Angle in?